### PR TITLE
Rework Sizes

### PR DESCRIPTION
**Pull request by Gaben1025**
## Changelog:
- Rebalanced sizes using maa buff/debuffs #729 

## How To Test:
- Play around and see if any specific culture has an unfair advantage over the other given their stats:

image

### CODE REVIEW

File: common/culture/innovations/wc_cultural_innovations.txt [hunk 1]
```diff
@@ -27,9 +27,17 @@ innovation_massive_soldiers = {
 		is_giant_culture_trigger = yes
 	}
 	modifier = {
-		levy_size = -0.5
-		advantage = 20
-		army_maintenance_mult = 0.15
+		men_at_arms_limit = -2
+		men_at_arms_maintenance = 0.12
+		skirmishers_damage_mult = 0.1
+		skirmishers_toughness_mult = 0.08
```
Comment (zumbak04) [inline common/culture/innovations/wc_cultural_innovations.txt:33]: Don't get why damage and toughness differ.

Comment (Gaben1025) [reply common/culture/innovations/wc_cultural_innovations.txt:33]: Toughness is a more valuable stat, would you prefer they be the same?

Also I will remove the maa limit like we talked about and replace it with more maintenance cost.

Comment (zumbak04) [reply common/culture/innovations/wc_cultural_innovations.txt]: > Toughness is a more valuable stat, would you prefer they be the same?

How it's more valuable? Yes.

File: common/culture/innovations/wc_cultural_innovations.txt [hunk 2]
```diff
@@ -63,9 +77,17 @@ innovation_small_soldiers = {
 		is_small_culture_trigger = yes
 	}
 	modifier = {
-		levy_size = 0.3
-		advantage = -10
-		army_maintenance_mult = -0.05
+		men_at_arms_limit = 1
```
Comment (zumbak04) [inline common/culture/innovations/wc_cultural_innovations.txt:80]: Is it intended?

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**